packages/web3/src/transaction/utils.test.ts

```diff
@@ -17,115 +17,295 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
 import { getSigner, getSigners } from '@alephium/web3-test'
-import { groupIndexOfTransaction } from './utils'
+import { groupIndexOfTransaction, waitForTxConfirmation } from './utils'
 import { ONE_ALPH, TOTAL_NUMBER_OF_GROUPS } from '../constants'
 import { bs58, hexToBinUnsafe } from '../utils'
 import { unsignedTxCodec } from '../codec'
 import { groupOfAddress } from '../address'
 import { blake2b } from 'blakejs'
 import { web3 } from '../'
 import { PrivateKeyWallet } from '@alephium/web3-wallet'
+import { node } from '../api'
 
 describe('transaction utils', () => {
   beforeAll(() => {
     web3.setCurrentNodeProvider('http://127.0.0.1:22973', undefined, fetch)
   })
 
-  it('should calculate the group of transfer transaction', async () => {
-    const signer1 = await getSigner(undefined, randomGroup())
-    const signer2 = await getSigner(undefined, randomGroup())
-    const fromAccount = await signer1.getSelectedAccount()
-    const toAccount = await signer2.getSelectedAccount()
+  describe('transaction confirmation', () => {
+    let originalProvider: any
```

Comment:
Question: What's the benefit of using `originalProvider` compared to before?

Reply:
my adding originalProvider was unnecessary, I see that now considering that the original tests did not modify the provider state. I  will have to change my implementations